jedi_vim.py

```diff
@@ -417,9 +416,6 @@ def rename():
                 vim.current.window.cursor = r.start_pos
                 vim_command('normal! cw%s' % replace)
 
-            result = new_buffer(window_path)
-            if not result:
-                return
             vim.current.window.cursor = cursor
```

Comment:
@blueyed I pretty sure that when I tried it with multiple files after @davidhalter comment i found that it was, or at least it seemed, broken. However I cannot reproduce it now. I just test it with a very simple example and it works fine. I created a `foo.py` file with 
```python
def f():
    pass
```
and a `bar.py` file with `from foo import f`. When I rename the `f()` function in a the `foo` file then it is also renamed in the `bar` file.

Thank you for looking into this! :)